raphtory/src/io/arrow/df_loaders.rs

```diff
@@ -2,20 +2,19 @@ use crate::{
     core::utils::errors::GraphError,
     db::api::{
         mutation::{internal::*, AdditionOps},
+        storage::storage_ops::GraphStorage,
         view::StaticGraphViewOps,
     },
     io::arrow::{dataframe::DFView, prop_handler::*},
     prelude::*,
 };
-use kdam::tqdm;
```

Comment:
Any reason to have removed the tqdm loaders throughout - they make the ingestion a lot clearer

Reply:
they were a python dependency inside the `io` module, I think they should only happen if used from python

Reply:
I believe they will also give you a loading bar from rust no? If you were using the parquet loader there etc. Where else would we put them if not here?